scripts/pull_github_pr.sh

```diff
@@ -9,33 +9,18 @@
 
 set -e
 
-gh_hosts=~/.config/gh/hosts.yml
-
-if [[ ( -z "$GITHUB_LOGIN" || -z "$GITHUB_TOKEN" ) && -f "$gh_hosts" ]]; then
-	GITHUB_LOGIN=$(awk '/user:/ { print $2 }' "$gh_hosts")
-	GITHUB_TOKEN=$(awk '/oauth_token:/ { print $2 }' "$gh_hosts")
-fi
-
 if [[ $# != 1 ]]; then
 	echo Please provide a github pull request number
 	exit 1
 fi
 
-for required in jq curl; do
+for required in jq gh; do
```

Comment:
Do we need extra steps to authenticate with `gh`, or will the previous way with `GITHUB_LOGIN` and `GITHUB_TOKEN` keep working?

Reply:
if `GITHUB_TOKEN` is available it will be used. other option are available as explained below
```
[yaronkaikov@london]~/git/test-pkg (fix_next_with_noARM)$ gh auth login --help
Authenticate with a GitHub host.

The default authentication mode is a web-based browser flow. After completion, an
authentication token will be stored securely in the system credential store.
If a credential store is not found or there is an issue using it gh will fallback
to writing the token to a plain text file. See `gh auth status` for its
stored location.

Alternatively, use `--with-token` to pass in a token on standard input.
The minimum required scopes for the token are: `repo`, `read:org`, and `gist`.

Alternatively, gh will use the authentication token found in environment variables.
This method is most suitable for "headless" use of gh such as in automation. See
`gh help environment` for more info.

```